test/integration/targets/win_psexec/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+- name: Download PSTools
+  win_get_url:
+    url: https://download.sysinternals.com/files/PSTools.zip
+    dest: C:\Windows\Temp\PSTools.zip
+    force: no
+
+- name: Create sysinternals directory
+  win_file:
+    path: C:\Program Files\sysinternals
```

Comment:
Why not unzip/run this in the tempdir in case someone actually has a C:\Program Files\sysinternals dir that wasn't created by the test?